pkgs/by-name/ma/massdns/package.nix

```
@@ -0,0 +1,33 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, nix-update-script
+}:
+stdenv.mkDerivation rec {
+  pname = "massdns";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "blechschmidt";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-hrnAg5ErPt93RV4zobRGVtcKt4aM2tC52r08T7+vRGc=";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" "PROJECT_FLAGS=-DMASSDNS_REVISION='\"v${version}\"'" ];
```

Split this over multiple lines just so that it looks nicer.
```suggestion
  makeFlags = [
    "PREFIX=$(out)"
    "PROJECT_FLAGS=-DMASSDNS_REVISION='\"v${version}\"'"
  ];
```